### PR TITLE
moving fetch2 library to jitpack

### LINKED ISSUE #2826: Jcenter dependency: androidx.tonyodev.fetch2
**Describe the bug**
We are using https://github.com/tonyofrancis/Fetch for downloading the file in our app. Its dependent on Jcenter.

I found that this library is being already moved to jitpack repo.

## https://github.com/tonyofrancis/Fetch/issues/586#issuecomment-954225519

Part of #2816

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes #2826

### CODE REVIEW

File: buildSrc/src/main/kotlin/plugin/AllProjectConfigurer.kt [hunk 1]
```diff
@@ -184,8 +184,7 @@ class AllProjectConfigurer {
       implementation(Libs.collection_ktx)
       implementation(Libs.butterknife)
       kapt(Libs.butterknife_compiler)
-      implementation(Libs.xfetch2)
-      implementation(Libs.xfetch2okhttp)
```
Comment (gouri-panda) [inline buildSrc/src/main/kotlin/plugin/AllProjectConfigurer.kt:188]: We need to `fetch2okhttp` for testing the download. How we are gonna implemented this? As the author didn't publish this module on Jitpack.

Comment (MohitMaliFtechiz) [reply buildSrc/src/main/kotlin/plugin/AllProjectConfigurer.kt:188]: yes the library on ```jitpack``` contain both ```xfetch2``` and ```xfetch2okhttp``` in one implementation.

File: buildSrc/src/main/kotlin/Versions.kt [hunk 2]
```diff
@@ -96,7 +96,7 @@ object Versions {
 
   const val barista: String = "2.7.1" // available: "3.7.0"
 
-  const val xfetch2: String = "3.1.6"
```
Comment (gouri-panda) [inline buildSrc/src/main/kotlin/Versions.kt:99]: We need to remove `xfetch2okhttp`  string as well.

Comment (MohitMaliFtechiz) [reply buildSrc/src/main/kotlin/Versions.kt:99]: done

**Review by gouri-panda — APPROVED** [review]

**Pull request merged**